package.json

```diff
@@ -7,22 +7,46 @@
     "start": "node index.js",
     "start:watch": "nodemon",
     "prestart:prod": "tsc",
-    "start:prod": "node dist/server.js"
+    "start:prod": "node dist/server.js",
+    "lint:check": "tslint ./src/**/*",
+    "lint:fix": "yarn-or-npm run lint:check -- --fix",
+    "check": "yarn-or-npm run lint:check && yarn-or-npm run prettier:check",
+    "check:fix": "yarn-or-npm run lint:fix; yarn-or-npm run prettier:fix",
+    "prettier:base":
+      "yarn-or-npm run prettier -- --single-quote --trailing-comma es5",
+    "prettier:base-files":
+      "yarn-or-npm run prettier:base -- './**/*.{js,ts,json}'",
+    "prettier:fix": "yarn-or-npm run prettier:base-files -- --write",
+    "prettier:check": "yarn-or-npm run prettier:base-files -- -l",
+    "precommit": "lint-staged",
+    "prepush": "yarn-or-npm run lint:check"
+  },
+  "lint-staged": {
+    "gitDir": "./",
+    "linters": {
+      "*.{js,ts,json}": ["yarn-or-npm run prettier:base -- -l"]
+    }
   },
   "dependencies": {
-    "@nestjs/common": "^4.0.0",
-    "@nestjs/core": "^4.0.0",
-    "@nestjs/microservices": "^4.0.0",
-    "@nestjs/testing": "^4.0.0",
-    "@nestjs/websockets": "^4.0.0",
-    "redis": "^2.7.1",
-    "reflect-metadata": "^0.1.10",
-    "rxjs": "^5.4.0",
-    "typescript": "^2.5.2"
+    "@nestjs/common": "4.0.0",
```

Comment:
why are you locking dependencies here? We're already on version 4.1.3.

Reply:
No reason, I'll fix that tonight